setup.py

```diff
@@ -99,7 +99,7 @@ class Test(TestCommand):
 
     def run(self):
         """Run tests."""
-        cmd = f"python3 -m pytest tests/ {self.get_args()}"
+        cmd = f"python3 -m pytest tests/ --cov-report term-missing {self.get_args()}"
```

Comment:
Was this `--cov-report term-missing` just temporary as you were fixing the tests?

Reply:
I like the idea of having this enabled by default. I get myself having to enable it from time to time to have more details on the coverage.

Reply:
Right, yes, let's keep it. @Alopalao, map another generic task on `kytos` to enable this `--cov-report term-missing` on kytos repo with checklist for the current NApps, and then folks who are working on a NApp they can enable it accordingly. Sometimes `term-missing` can be a bit verbose, but it's worth it (but when it's verbose it's already due to a lack of good cov, so that's another incentive to go ahead and have a better cov in the first place). From what I've seen it doesn't break the column layout, so let's enable it.